src/Common/src/Interop/Unix/System.Native/Interop.FnMatch.cs

```diff
@@ -0,0 +1,21 @@
+using System;
+using System.Runtime.InteropServices;
+
+internal static partial class Interop
+{
+    internal static partial class Sys
+    {
+        internal const int FNM_NOMATCH = 1; // constant from fnmatch.h
```

Comment:
It is not currently used, but since the documentation calls it out as a very specific return value, I thought it would be useful to declare in case any future code change needed it. I can remove it if you would like